### PR TITLE
feat: remove nextBatch

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Next batch is removed

# To Test

1. Should work as usual, this was not easily reachable anyway

### CODE REVIEW

File: apps/cowswap-frontend/src/common/hooks/orderProgressBarV2/types.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@ export type OrdersProgressBarState = Record<string, OrderProgressBarState>
 
 export type OrdersProgressBarCountdown = Record<string, number | null>
 
-type happyPath = 'initial' | 'solving' | 'solved' | 'executing' | 'finished'
-type errorFlow = 'nextBatch' | 'delayed' | 'unfillable' | 'submissionFailed'
+type happyPath = 'initial' | 'solving' | 'executing' | 'finished'
+type errorFlow = 'delayed' | 'solved' | 'unfillable' | 'submissionFailed'
```
Comment (anxolin) [inline apps/cowswap-frontend/src/common/hooks/orderProgressBarV2/types.ts:19]: not coming from this PR, but shouldn't the types be capitalised?

Comment (alfetopito) [reply apps/cowswap-frontend/src/common/hooks/orderProgressBarV2/types.ts:19]: Nope

Comment (alfetopito) [reply apps/cowswap-frontend/src/common/hooks/orderProgressBarV2/types.ts:19]: They are sort of inner logical division. The exported one is the combination of all of them.

**Review by elena-zh — APPROVED** [review]
Might not be related to this PR, but noticed that several times countdown displaying finished when 3 sec were left to the end

**Comment by alfetopito** [comment]
> Might not be related to this PR, but noticed that several times countdown displaying finished when 3 sec were left to the end

Unrelated to these changes. Will check in another PR

**Pull request merged**